src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -192,12 +200,17 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
-    loadLearningGoals() {
-        forkJoin([this.learningGoalService.getAllForCourse(this.courseId), this.learningGoalService.getAllPrerequisitesForCourse(this.courseId)]).subscribe({
-            next: ([learningGoals, prerequisites]) => {
+    loadLearningGoalsAndTutorialGroups() {
+        forkJoin([
+            this.learningGoalService.getAllForCourse(this.courseId),
+            this.learningGoalService.getAllPrerequisitesForCourse(this.courseId),
+            this.tutorialGroupService.getAllOfCourse(this.courseId),
```

Comment:
Maybe make the method naming consistent with the other functions?
```suggestion
            this.tutorialGroupService.getAllForCourse(this.courseId),
```

Reply:
Done! Changed in 404a61f9e625e0e40fb810485d800baaea28b685